src/holodeck/util.py

```diff
@@ -93,3 +95,66 @@ def human_readable_size(size_bytes):
     power = math.pow(1024, base)
     size = round(size_bytes / power, 2)
     return "%s %s" % (size, size_name[base])
+
+
+def draw_line(env, start, end, color=None, thickness=10.0):
+    """Draws a debug line in the world
+
+    Args:
+        env (:class:`~holodeck.environments.HolodeckEnvironment`): Environment to draw in.
+        start (:obj:`list` of :obj:`float`): The start ``[x, y, z]`` location of the line.
+            (see :ref:`coordinate-system`)
+        end (:obj:`list` of :obj:`float`): The end ``[x, y, z]`` location of the line
+        color (:obj:`list``): ``[r, g, b]`` color value
+        thickness (:obj:`float`): thickness of the line
+    """
+    color = [255, 0, 0] if color is None else color
+    command_to_send = DebugDrawCommand(0, start, end, color, thickness)
+    env._enqueue_command(command_to_send)
```

Comment:
Just to follow the Python convention, you may want to avoid calling class methods with a leading underscore since those are "non-public". I wonder if it's appropriate to make  `_enqueue_command` public. Thoughts @jaydenmilne , @MaxRobinsonTheGreat?

Reply:
I talked about this specific issue with @jaydenmilne, and he recommended just referencing `_enqueue_command` this way. I could see that this might not be a big issue for debug commands, but I wouldn't be opposed to making `_enqueue_command` publicly visible either.

Reply:
I guess you could think of "private" as "internal implementation detail subject to change whenever" in Python. Honestly there is a lot of stuff that should be "private" (like most commands I think), but I don't loose too much sleep over it

Reply:
@daniekpo @jaydenmilne Coming from statically typed languages with visibility restrictions, this kind of thing does feel a little bit gross to me, but I'd be okay with doing it this way now. I would probably make `_enqueue_command` public if it were on my own codebase, but I'm thinking the real issue would be if we started _consistently_ accessing it from outside `HolodeckEnvironment`.

Reply:
Yeah, it's really not that big of a deal. We can just keep it this way.